ttnn/ttnn/operations/pool.py

```diff
@@ -92,17 +92,17 @@ def __init__(
         stride: Union[int, Tuple[int, int]] = 1,
         padding: Union[int, Tuple[int, int]] = 0,
         dilation: Union[int, Tuple[int, int]] = 1,
-        dtype: ttnn.DataType = None,
+        dtype: Optional[ttnn.DataType] = None,
```

Comment:
@esmalTT @mywoodstock when is this py op gonna be removed from ttnn?

Reply:
Once unet is ported to using the new api. Thats the main dependency blocking removal of old version